.github/workflows/PR-Demo-Comment-with-react.yml

```diff
@@ -223,7 +225,8 @@ jobs:
                 - /stirling/PR-${{ needs.check-comment.outputs.pr_number }}/config:/configs:rw
                 - /stirling/PR-${{ needs.check-comment.outputs.pr_number }}/logs:/logs:rw
               environment:
-                DOCKER_ENABLE_SECURITY: "${DOCKER_SECURITY}"
+                DOCKER_ENABLE_SECURITY: "${DOCKER_SECURITY}" # todo: change DOCKER_SECURITY?
```

Comment:
Lets remove DOCKER_ENABLE_SECURITY from all Config, i want it maintained for old users so migration is easy, but I don't want new users to use something so badly named haha